File: Modul5/Bahasa_Python/PRAK504-2310817210012-Nazmi_Hakim.py
def reverse(angka):
    dibalik = 0
    while angka > 0:
        dibalik = dibalik * 10 + angka % 10
        angka //= 10
    return dibalik

A, B = map(int, input().split())
A = reverse(A)
B = reverse(B)
C = A + B
print(reverse(C))

File: Modul5/Bahasa_C/PRAK504-2310817210012-Nazmi_Hakim.c
#include <stdio.h>
int reverse(int angka){
    int dibalik = 0;
    while (angka > 0){
        dibalik = dibalik * 10 + angka % 10;
        angka /= 10;
}
return dibalik;
}
int main() {
 int A, B;
 scanf("%d %d",&A,&B);
 A=reverse(A);
 B=reverse(B);
 int C = A+B;
 printf("%d",reverse(C));
}